ppb/systems/sound.py

```diff
@@ -28,6 +28,7 @@ class Sound(assetlib.Asset):
     # This is wrapping a ctypes.POINTER(Mix_Chunk)
 
     def background_parse(self, data):
+        print("Sound.background_parse", repr(self), flush=True)
```

Comment:
Did you mean to leave the print debugging here?

Reply:
Not at all